en/04_Changelogs/5.0.0.md

```diff
@@ -146,6 +186,7 @@ This release includes a number of bug fixes to improve a broad range of areas. C
 ## Environment variable changes
 
 - `SS_MANIFESTCACHE` can no longer use the now removed symfony/cache 4.x "Simple" cache classes e.g. `Symfony\Component\Cache\Simple\PhpFilesCache`. Instead use the corresponding "Adapter" class e.g. `Symfony\Component\Cache\Adapter\PhpFilesAdapter`.
+- `APP_SMTP_USERNAME` and `APP_SMTP_PASSWORD` no longer have any affect. Use a `MAILER_DSN` environment variable instead to configure SMTP email
```

Comment:
```suggestion
- `APP_SMTP_USERNAME` and `APP_SMTP_PASSWORD` have been removed. Use a `MAILER_DSN` environment variable instead to configure SMTP email (see [the email documentation](/developer_guides/email/) for more details).
```
We may as well use stronger language here.

Reply:
Updated